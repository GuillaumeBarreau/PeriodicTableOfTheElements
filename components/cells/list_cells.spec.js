import { shallowMount } from '@vue/test-utils'
import cmp from '~/components/cells/list_cells'

describe('Testing component List_cells', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(cmp, {
    })
  })

  it('snapshot testing', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
