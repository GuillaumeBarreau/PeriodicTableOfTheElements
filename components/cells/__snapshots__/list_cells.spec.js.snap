// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component List_cells snapshot testing 1`] = `"<ul class=\\"content--elements\\"></ul>"`;
